### PR TITLE
added redis conf

### PR DESCRIPTION
**Pull request by shimonewman**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -640,7 +640,9 @@ services:
   #     - SMARTHOST_PORT=587
   
   redis:
-    image: redis:6.2-alpine
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
```
Comment (christyjacob4) [inline docker-compose.yml:645]:
```suggestion
      dockerfile: redis/Dockerfile
```

Maybe its better to keep all the redis related things in a folder called `redis/`

that way we would have `redis/redis.conf` and `redis/Dockerfile`

Comment (christyjacob4) [inline docker-compose.yml:645]:
```suggestion
      dockerfile: redis/Dockerfile
```

Maybe its better to keep all the redis related things in a folder called `redis/`

that way we would have `redis/redis.conf` and `redis/Dockerfile`

**Review by eldadfux — APPROVED** [review]
Nice, I like it, very elegant.

**Review by TorstenDittmann — COMMENTED** [review]
Can you add those cahnges to `app/views/install/compose.phtml` as well?

**Review by christyjacob4 — APPROVED** [review]
Much better than the original 👌

**Pull request merged**